Models/ChatCommunicator.cs

```diff
@@ -0,0 +1,79 @@
+using Tinker_Back.Models;
+using System.Linq;
+namespace Tinker_Back
+{
+    public class ChatCommunicator
+    {
+        private TinkerDbContext _context;
+        public ChatCommunicator(TinkerDbContext dbContext)
+        {
+            _context = dbContext;
+        }
+        private void ProcessChatIfExist(int chatID, Action<Chat> func)
+        {
+            Chat? chat = _context.Chats.Find(chatID);
+            if(chat == null)
+            {
+                func(chat);
+            }
+            _context.SaveChanges();
+        }
+        private void ProcessPinChat(int chatID, bool pinned)
+        {
+            ProcessChatIfExist(chatID, (Chat chat) => chat.IsPinned = pinned);
+        }
+        public void CreateChat(string name, string description)
+        {
+            Chat chat = new Chat(name, description);
+            _context.Chats.Add(chat);
+            _context.SaveChanges();
+        }
+        public void DeleteChat(int chatID)
+        {
+            ProcessChatIfExist(chatID, (Chat chat) => _context.Chats.Remove(chat));
+        }
+        public void ChangeDescription(int chatID, string newText)
```

Comment:
In other functions naming is `ChangeName (..., string newName)`
But here 
`ChangeDescription (..., string newText)`
probably it will be better to name it 

`ChangeDescription (..., string newDescription)`